src/app/home-page/new-dashboard/new-dashboard.component.html

```diff
@@ -5,10 +5,30 @@
     <app-header>My Dockstore: Dashboard</app-header>
     <app-news-notifications class="w-100"></app-news-notifications>
     <div fxLayout="column" fxLayoutAlign="start start" fxLayoutGap="2.5rem" class="p-4">
+      <!-- Notebooks Warning -->
+      <mat-card class="alert alert-info mat-elevation-z w-100" *ngIf="Dockstore.FEATURES.enableNotebooks" data-cy="notebooks-warning">
+        <mat-icon>warning</mat-icon>
+        <strong> Notice:</strong> This is an early access feature. Consider this a prototype that may be more likely to have breaking
+        changes going forward than other Dockstore features.
+      </mat-card>
+      <!-- End of Notebooks Warning -->
       <div fxFlex fxLayout fxLayout.lt-md="column" fxLayoutGap="2.5rem" class="w-100">
         <app-entry-box fxFlex="33" fxFlex.lt-md="100" class="entry-holder" entryType="WORKFLOW"></app-entry-box>
         <app-entry-box fxFlex="33" fxFlex.lt-md="100" class="entry-holder" entryType="TOOL"></app-entry-box>
-        <app-entry-box fxFlex="33" fxFlex.lt-md="100" class="entry-holder" entryType="SERVICE"></app-entry-box>
+        <app-entry-box
+          fxFlex="33"
+          fxFlex.lt-md="100"
+          class="entry-holder"
+          entryType="SERVICE"
+          *ngIf="Dockstore.FEATURES.enableNotebooks === false"
```

Comment:
Is this on purpose, that `entryType="SERVICE"` is dependent on the the enableNotebooks flag not being set?

Reply:
Yep, so we display either one or the other, but we'll default to displaying services:

![image](https://user-images.githubusercontent.com/97123241/228323438-4a5bd872-2e3e-409d-81e1-258c9bad0255.png)
![image](https://user-images.githubusercontent.com/97123241/228323512-16de6cbd-4435-4764-b00a-fe3822803e77.png)

Glad you brought this up, realized I can simplify the logic here :) `entryType="{{ Dockstore.FEATURES.enableNotebooks ? 'NOTEBOOK' : 'SERVICE' }}"`